wifi/src/Station.cpp

```diff
@@ -26,28 +30,38 @@ namespace sdk {
         }
 
         res Station::getStatus() {
-            return Ok(ComponentStatus::RUNNING);
+            return ComponentStatus::RUNNING;
         }
 
         res Station::run() {
-            return Ok(ComponentStatus::RUNNING);
+            return ComponentStatus::RUNNING;
         }
 
         res Station::stop() {
-            wifi_mode_t currentMode  = WIFI_MODE_NULL;
-            esp_err_t   err          = esp_wifi_get_mode(&currentMode);
-            if (err == ESP_ERR_WIFI_NOT_INIT)
-                ESP_LOGW(TAG, "Wifi is not running, returning");
-            else if (err == ESP_OK && currentMode == WIFI_MODE_STA) {
-                RETURN_ON_ERR_MSG(esp_wifi_stop(), "esp_wifi_stop: ");
-            } else if (err == ESP_OK && currentMode == WIFI_MODE_APSTA)
-                RETURN_ON_ERR_MSG(esp_wifi_set_mode(WIFI_MODE_AP), "esp_wifi_set_mode: ");
-            return Ok(ComponentStatus::STOPPED);
+            wifi_mode_t currentMode = WIFI_MODE_NULL;
+            esp_err_t   err         = esp_wifi_get_mode(&currentMode);
+            if (err == ESP_ERR_WIFI_NOT_INIT) {
+                ESP_LOGD(TAG, "Wifi is not running, returning");
```

Comment:
```suggestion
                ESP_LOGW(TAG, "Wifi is not running, returning");
```
Any reasoning behind changing this to debug?